test/e2e/lib/test-utils.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+
+const BASE_PATH = process.env.TEMP_ASSET_PATH || path.join( __dirname, '..' );
+
+export const generatePath = ( name ) => path.join( BASE_PATH, name );
```

Comment:
This helper ensures the logs are stored together with the reports, so it is easy to identify which tests they belong to.

Right now that folder is named something like`build-magellan-76e89eab6efe5_89ddf098a7d10__temp_assets`. In a future PR we can change that to use the suite name.